run-build.sh

```diff
@@ -30,18 +30,17 @@ else
 fi
 
 # Node version
-source /.nvm/nvm.sh
-export NODE_VERSION=0.12.2
-if [ -f .nvmrc ]
-then
-	if nvm use; then
-		echo "Set node from .nvmrc"
-		export NODE_VERSION=$(cat .nvmrc)
-	else
-		echo "Error setting node version from .nvmrc file. Unsupported version?"
-		echo "Will use default version ($NODE_VERSION)"
-	fi
+source $HOME/.nvm/nvm.sh
+export NODE_VERSION=4
+if [ -f .nvmrc ]; then
+  nvm install $(cat .nvmrc)
```

Comment:
When changing nvm version, your global node_modules aren't accessible from the previous version. So grunt, bower, etc. that you install [in the Dockerfile](https://github.com/netlify/build-image/blob/ccf8dbb8a5c25e58154cd61f0b56534670751242/Dockerfile#L111-L112) is not accessible, since the link to your node/bin dir is changed as well.